drivers/gles2/rasterizer_scene_gles2.cpp

```diff
@@ -2421,6 +2421,14 @@ void RasterizerSceneGLES2::_render_render_list(RenderList::Element **p_elements,
 						}
 
 					} break;
+					case RasterizerStorageGLES2::Shader::Spatial::BLEND_MODE_PMALPHA: {
+						glBlendEquation(GL_FUNC_ADD);
+						if (storage->frame.current_rt && storage->frame.current_rt->flags[RasterizerStorage::RENDER_TARGET_TRANSPARENT]) {
+							glBlendFuncSeparate(GL_ONE, GL_ONE_MINUS_SRC_ALPHA, GL_ONE, GL_ONE_MINUS_SRC_ALPHA);
+						} else {
+							glBlendFunc(GL_ONE, GL_ONE_MINUS_SRC_ALPHA);
```

Comment:
These are functionally equivalent. In either case source is multiplied by GL_ONE and destination is multiplied by GL_ONE_MINUS_SRC_ALPHA

Reply:
That's what I thought, but I wasn't 100% sure, so I did it like this to be safe.